specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/examples/VirtualMachineRunCommand.json

```diff
@@ -4,7 +4,7 @@
     "resourceGroupName": "crptestar98131",
     "vmName": "vm3036",
     "$top": "1",
-    "api-version": "2017-12-01",
+    "api-version": "2018-06-01",
```

Comment:
Was this intended at that time, or it's just found as typo errors till now?

Reply:
This file was copied from 2017-12-01 to 2018-06-01 folder with no reasons, since the Swagger which was mentioning it didn't exist. This is correct then the correct content for the file.

Reply:
But it may be intended by the service API team. What if the RunCommand APIs (for some reason) don't work with this new API version? Swagger spec correctness is important, but we can't also expose something here that the service API is not ready to expose.

@hyonholee Can you help involve Atanas to help confirm? Thanks.

Reply:
Please see prev version example when the API has changes:
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/examples/VirtualMachineRunCommand.json

The output: [ ] become wrapped in json object 
output: {value: [ ] }

Reply:
changed